front/src/applications/operationalStudies/components/SimulationResults/SimulationResultsMap.tsx

```diff
@@ -142,9 +142,10 @@ const Map: FC<MapProps> = ({ setExtViewport }) => {
     const concernedTrains: InterpoledTrain[] = [];
     simulation.trains.forEach((train, idx: number) => {
       const key = getRegimeKey(train.id);
```

Comment:
might as well do that everywhere :)
```suggestion
      const baseOrEco = getRegimeKey(train.id);
```

Reply:
Done :)